yt_dlp/extractor/bfmtv.py

```diff
@@ -102,18 +129,14 @@ class BFMTVArticleIE(BFMTVBaseIE):
         },
     }]
 
+    def _entries(self, webpage):
+        yield from traverse_obj(
+            re.findall(self._VIDEO_BLOCK_REGEX, webpage), (..., {self._extract_video}, filter, all))
```

Comment:
This makes only little sense since `traverse_obj` returns a list, always. Probably bad design, but compat and cannot be changed